### ISSUE TITLE
Let windows users have the full color experience

### ISSUE THREAD

**Issue by jcwillox** [sharkdp/hyperfine#427]
I don't believe anyone's asked this before but I can't see a good reason to downgrade windows users from `full` to `color` especially when they explicitly specify it.

https://github.com/sharkdp/hyperfine/blob/1027940e3a8dfd6a1b284d4d2ce1a5f6d1857359/src/main.rs#L219-L222

In fact, you may consider making windows default to `full` as Windows Terminal has support for true colour and recently added bold support in the previews. As well the default windows conhost now has support for ANSI colours, although you will need to enable support in your program or enable it globally via the registry. That last bit isn't as important as I'd assume most people wouldn't be using hyperfine with conhost.

https://stackoverflow.com/a/51681675/12346307 enabling ANSI support for conhost
https://github.com/mattn/go-colorable a library I've used before that handles enabling ANSI support, not sure if rust has an equivalent.

**Comment by sharkdp** [comment]
I agree, let's change this. Related (old) discussion: https://github.com/sharkdp/hyperfine/pull/37

**Comment by sharkdp** [comment]
@jcwillox Could you let me know if this works in https://github.com/sharkdp/hyperfine/releases/tag/v1.12.0 ?

**Comment by jcwillox** [comment]
Yessss, fantastic colours are working correctly, thanks!

Side note the latest update broke the scoop manifest, as the binary is now inside a directory, I've already published a PR to fix this though https://github.com/ScoopInstaller/Main/pull/2773.
